httpbakery/visitor_test.go

```diff
@@ -147,6 +148,29 @@ func (*VisitorSuite) TestUserInteractionFallback(c *gc.C) {
 	c.Assert(called, gc.Equals, 1)
 }
 
+func (*VisitorSuite) TestMultiVisitorVisitorError(c *gc.C) {
+	srv := httptest.NewServer(http.HandlerFunc(func(w http.ResponseWriter, req *http.Request) {
+		w.Header().Set("Content-Type", "application/json")
+		fmt.Fprint(w, `{"method": "http://somewhere/something"}`)
+	}))
+	defer srv.Close()
+
+	testError := errgo.New("test error")
+	called := 0
+	v := httpbakery.NewMultiVisitor(
+		visitorFunc(func(_ *httpbakery.Client, m map[string]*url.URL) error {
+			c.Check(m["method"], gc.NotNil)
+			called++
```

Comment:
No real need to check called here - if we get a testError out of VisitWebPage, there's nowhere else it could've come from. Similarly for checking m["method"] - we're not testing that aspect of MultiVisitor here.